ouroboros-consensus-byron/ouroboros-consensus-byron.cabal

```diff
@@ -61,6 +61,7 @@ library
                      , formatting        >=6.3   && <6.4
                      , mtl               >=2.2   && <2.3
                      , serialise         >=0.2   && <0.3
+                     , nothunks
```

Comment:
FYI, we don't need bounds on `nothunks` as we are compatible with any *currently released* version (0.1.0.0 and 0.1.1.0). When ["Derive NoThunks for Void"](https://github.com/input-output-hk/nothunks/commit/6fd1cf7a3c77607ba190881b48b219dd50dc5ef7) becomes part of a new release on Hackage and our index-state is updated to a date *after* the release, *then* we'll need to remove our orphan `NoThunks Void` instance from `Ouroboros.Consensus.Util.Orphans` and set that release as the new lower bound.